styles.css

```diff
@@ -684,3 +712,413 @@ footer {
+    left: 433px;
+    top: 62px;
+    margin-left: 362px;
+  }
+
+  .form-list {
+    display: flex;
+    flex-direction: row;
+  }
+
+  #last-name {
+    display: inline;
+  }
+
+  .footer-line {
+    display: none;
+  }
+
+  #email-id {
+    width: 92%;
+  }
+
+  #message {
+    position: relative;
+    left: 12px;
+    right: 12px;
+    top: 12px;
+    bottom: 12px;
+    font-family: "Inter", sans-serif;
+    font-weight: 400;
+    font-size: 15px;
+    line-height: 24px;
+    color: #3c3a39;
+    font-style: italic;
+    width: 95%;
+  }
+
+  .button {
+    align-items: center;
+    display: flex;
+    flex-direction: row;
+    margin-top: 24px;
+    padding: 0 0 0 9px;
+  }
+
+  .footer-top {
+    position: absolute;
+    height: 1px;
+    width: 375px;
+    left: 0;
+    top: 0;
+    border-radius: 0;
+    background: #dfe1e6;
+  }
+
+  footer {
+    margin: 61px 0;
+    padding: 12px 6px;
+  }
+
+  /* ----------------------grid work section -------destop----------------- */
+
+  .main-1st-Title {
+    display: grid;
+    align-items: center;
+    justify-content: start;
+    width: 100%;
+    font-family: "Crete Round", sans-serif;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 40px;
+    line-height: 52px;
+    text-align: center;
+    color: black;
+  }
+
+  h2 {
+    width: 100%;
+    height: 81px;
+    font-family: "Crete Round", sans-serif;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 32px;
+    line-height: 44px;
+    color: #fff;
+    margin-top: 0;
+    padding-top: 0;
+  }
+
+  .professional-art {
+    display: none;
+  }
+
+  .professional-art1 {
+    display: grid;
+    grid-template-columns: 1fr 1fr 1fr;
+    gap: 20px;
+    margin-top: 60px;
+    margin-bottom: 100px;
+    padding: 0 5%;
+  }
+
+  .details1 h2 {
+    padding-left: 25px;
+  }
+
+  .web-padd {
+    padding-top: 15px;
+  }
+
+  .details1 {
+    background-image: url(images/availabilty.png);
+    background-repeat: no-repeat;
+    background-size: cover;
+    width: 100%;
+    color: white;
+    flex-wrap: wrap;
+  }
+
+  .details2 {
+    background-image: url(images/webportfolio.png);
+    background-repeat: no-repeat;
+    background-size: cover;
+    width: 100%;
+    color: white;
+    flex-wrap: wrap;
+  }
+
+  .detailss {
+    background-image: url(images/1st.png);
+    background-repeat: no-repeat;
+    background-size: cover;
+    width: 100%;
+    color: white;
+    flex-wrap: wrap;
+  }
+
+  .see-project-btn1 {
+    font-family: "Inter", sans-serif;
+    font-style: normal;
+    font-weight: 700;
+    font-size: 17px;
+    position: relative;
+    width: 100%;
+    height: 48px;
+    left: 0;
+    top: 0;
+    background: #ff6b00;
+    color: #fff;
+    border: #fff;
+  }
+
+  .see-project-btn {
+    display: none;
+  }
+
+  .programs1,
+  .data-arc5,
+  .prof-1 {
+    display: none;
+  }
+
+  .line-flex {
+    display: flex;
+    align-items: center;
+    margin: 114px 0;
+    text-align: start;
+  }
```

Comment:
- [ ] Please I kindly suggest sticky to the [Figma Design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=34-1278&t=m82U1CnKG1ogD0DD-0)

_**REQUIRED FIGMA DESIGN**_

![Screenshot (131)](https://user-images.githubusercontent.com/78104544/228996556-4148969e-115e-4797-8b20-bab2c0c59817.png)


_**YOUR DESIGN**_

- [ ] Please I kindly suggest implementing changes on the `.line-flex` class to close the gap to fit the Figma design. 😄 

![Screenshot (129)](https://user-images.githubusercontent.com/78104544/228996630-897bf0fa-2dc8-4990-93a1-c590e38dd474.jpg)

Reply:
resolved